README.md

```diff
@@ -10,16 +10,25 @@ You can use it as follows:
 
         git clone https://github.com/gap-packages/PackageMaker
 
-2. Start GAP, load the `PackageMaker` package and run the package wizard:
+Or, you may want to install `PackageMaker` using the [PackageManager](https://github.com/gap-packages/PackageManager) GAP package by typing `LoadPackage("PackageManager"); InstallPackage("PackageMaker");` in GAP.
+
+        InstallPackage("PackageMaker");
```

Comment:
This won't work right now because PackageManager won't know where to find this package. Perhaps like this, at least for now:
```suggestion
Alternatively you could install `PackageMaker` using the [PackageManager](https://github.com/gap-packages/PackageManager) GAP package by entering these commands in GAP:

        LoadPackage("PackageManager");
        InstallPackage("https://github.com/gap-packages/PackageMaker");
```

Reply:
@fingolfin Why isn't it working? I have been using the package manager to install packages recently, and it works well. As Michel mentioned in his talk yesterday, he is currently fetching data from the `csv`, but he plans to change the structure and fetch all the data in one go using a caching mechanism from a new `json` file, which already exists.